cli/src/cli.rs

```diff
@@ -1530,11 +1530,6 @@ fn process_transfer(
 ) -> ProcessResult {
     let from = config.signers[from];
 
-    check_unique_pubkeys(
-        (&from.pubkey(), "cli keypair".to_string()),
-        (to, "to".to_string()),
-    )?;
-
```

Comment:
I think we can remove this check from `process_pay` as well.

Reply:
About time we rip out all things Budget. All that code can have a new home in the SPL. We can add a new `solana-budget` CLI as well.